Examples/BookingSystem.AspNetFramework/Web.config

```diff
@@ -5,8 +5,8 @@
   -->
```

Comment:
not sure why this file has changed

Reply:
Likely the difference between Mac and Windows Visual Studio